frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/enterprise.embedding.unit.spec.tsx

```diff
@@ -58,6 +58,45 @@ describe("[EE, no token] embedding settings", () => {
       });
     });
 
+    describe("interactive embedding", () => {
```

Comment:
The changes are exactly the same as the `common.embedding.unit.spec.tsx`

Reply:
well yeah, I hope we can do that too. The thing is that since these setups are different we need to duplicate most of the tests, otherwise, we could have shared tests in one file and export them into different files e.g. for common scenarios then export those tests and put them in `common*`, `enterprise*`, `premium*`.

This is worth another RFC. I'll try to open one when I have time.